deploy/02-sa.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
fix the name on this, we don't want any overlap on the numbering.  The numbering should control the order these resources are created in  (so you also probably want to create the SA before the rolebindings, though strictly speaking i don't think it matters)

Reply:
yeah in comparing with some other repos I saw where several cases where steps that could happen in either order had the same number

i.e. the SA and the rolebinding where as you noted they can occur in either order (my manual testing saw that)

but sure, I can be explicit, and will put the SA before the rolebinding